### ISSUE TITLE
Suggestion database entry for `Base.System.File.read` is missing `SuggestionEntryArgument` for `path`

### ISSUE THREAD

**Issue by MichaelMauderer** [enso-org/enso#1320]
### General Summary
The suggestion database entry for a `File.read` call on the IDE side looks like this:

```
SuggestionsDatabaseEntry { 
   id: 698, 
   suggestion: Method { 
        external_id: None, 
        name: "read", 
        module: "Base.System.File", 
        arguments: [
            SuggestionEntryArgument { name: "this", 
                                              repr_type: "File", 
                                              is_suspended: false, 
                                              has_default: false, 
                                              default_value: None 
            }
        ],
       self_type: "File", 
       return_type: "File_Error", 
       documentation: Some(" Reads the whole file into a `Text`, assuming UTF-8 content encoding.") 
   } 
}
```

The corresponding Enso function in `std-lib/Base/src/System/File.enso`  looks like this:
```
read : Text -> Text
read path = [...]
```

So, it seems there should be a `SuggestionEntryArgument` for the `path` argument. 

This appears to be the root cause of https://github.com/enso-org/ide/issues/998

### Steps to Reproduce

1. Launch the Enso project manager.
2. Launch IDE
3. Observe response on `search/getSuggestionsDatabase` rpc.

### Expected Result
The entry should contain a ` SuggestionEntryArgument { name: "path", [...] } `.

### Actual Result
The entry does not contain a ` SuggestionEntryArgument { name: "path", [...] } `.

### Enso Version

```Enso Project Manager
Version:    0.1.0
Built with: scala-2.13.3 for GraalVM 20.2.0
Built from: main* @ d608e21b39969b97ba478c40c23592fe432cccf5
Running on: OpenJDK 64-Bit Server VM, GraalVM Community, JDK 11.0.8+10-jvmci-20.2-b03
            Linux 5.8.0-7630-generic (amd64)
```

**Comment by MichaelMauderer** [comment]
Note: this works for `File.read_csv` as expected.

**Comment by 4e6** [comment]
Caused by the #1097

`Base.System.File` module contains two `read` methods (defined on the `File` atom and the `File` module). Because the types are not qualified, they look like duplicates, and only the first is stored in the suggestion database

**Comment by 4e6** [comment]
Will be fixed by #1097
